### PR TITLE
JENKINS-32265 - Added "Slave Version" using `Launcher.VERSION` to master node

### PR DESCRIPTION
**Pull request by Evildethow**
@reviewbybees

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
:bee: . BTW Slave versions for remote slaves are much more important

File: src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java [hunk 1]
```diff
@@ -803,6 +804,7 @@ private String getLabelString(Node n) {
                     .replaceAll("`", "&#96;") + "`");
             out.println("      - Labels:         " + getLabelString(Jenkins.getInstance()));
             out.println("      - Usage:          `" + Jenkins.getInstance().getMode() + "`");
+            out.println("      - Slave Version:  " + Launcher.VERSION);
```
Comment (oleg-nenashev) [inline src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java]: Maybe the wording should be equal to slaves (they use "       - Version: ")

**Comment by varmenise** [comment]
:bee:

**Comment by christ66** [comment]
:bee:

**Pull request merged**